wandb/proto/wandb_telemetry_pb2.pyi

```diff
@@ -1,129 +1,112 @@
+global___Imports = Imports
+
+class Feature(google.protobuf.message.Message):
+    DESCRIPTOR: google.protobuf.descriptor.Descriptor = ...
+    watch: builtins.bool = ...
+    finish: builtins.bool = ...
+    save: builtins.bool = ...
+    offline: builtins.bool = ...
+    resumed: builtins.bool = ...
+    grpc: builtins.bool = ...
 
     def __init__(self,
         *,
-        watch : typing___Optional[builtin___bool] = None,
-        finish : typing___Optional[builtin___bool] = None,
-        save : typing___Optional[builtin___bool] = None,
-        offline : typing___Optional[builtin___bool] = None,
-        resumed : typing___Optional[builtin___bool] = None,
-        grpc : typing___Optional[builtin___bool] = None,
+        watch : builtins.bool = ...,
+        finish : builtins.bool = ...,
+        save : builtins.bool = ...,
+        offline : builtins.bool = ...,
+        resumed : builtins.bool = ...,
+        grpc : builtins.bool = ...,
         ) -> None: ...
-    def ClearField(self, field_name: typing_extensions___Literal[u"finish",b"finish",u"grpc",b"grpc",u"offline",b"offline",u"resumed",b"resumed",u"save",b"save",u"watch",b"watch"]) -> None: ...
-type___Feature = Feature
+    def ClearField(self, field_name: typing_extensions.Literal[u"finish",b"finish",u"grpc",b"grpc",u"offline",b"offline",u"resumed",b"resumed",u"save",b"save",u"watch",b"watch"]) -> None: ...
+global___Feature = Feature
 
-class Env(google___protobuf___message___Message):
-    DESCRIPTOR: google___protobuf___descriptor___Descriptor = ...
-    jupyter: builtin___bool = ...
-    kaggle: builtin___bool = ...
-    windows: builtin___bool = ...
-    m1_gpu: builtin___bool = ...
+class Env(google.protobuf.message.Message):
+    DESCRIPTOR: google.protobuf.descriptor.Descriptor = ...
+    jupyter: builtins.bool = ...
+    kaggle: builtins.bool = ...
+    windows: builtins.bool = ...
+    m1_gpu: builtins.bool = ...
 
     def __init__(self,
         *,
-        jupyter : typing___Optional[builtin___bool] = None,
-        kaggle : typing___Optional[builtin___bool] = None,
-        windows : typing___Optional[builtin___bool] = None,
-        m1_gpu : typing___Optional[builtin___bool] = None,
+        jupyter : builtins.bool = ...,
+        kaggle : builtins.bool = ...,
+        windows : builtins.bool = ...,
+        m1_gpu : builtins.bool = ...,
         ) -> None: ...
-    def ClearField(self, field_name: typing_extensions___Literal[u"jupyter",b"jupyter",u"kaggle",b"kaggle",u"m1_gpu",b"m1_gpu",u"windows",b"windows"]) -> None: ...
-type___Env = Env
+    def ClearField(self, field_name: typing_extensions.Literal[u"jupyter",b"jupyter",u"kaggle",b"kaggle",u"m1_gpu",b"m1_gpu",u"windows",b"windows"]) -> None: ...
+global___Env = Env
```

Comment:
Hmmm, I don't think these proto bugs should have changed...

Reply:
hmm.. i pinned the protobuf libraray during `make proto` but i didnt do that for the proto typing library, so thats why it changed (i think)

Reply:
I've pinned the protobuf typing library to 1.23